documentation/decisions/QuestionPoll.md

```diff
@@ -0,0 +1,28 @@
+## Question Poll / reguläre Abstimmung
+
+*In diesem Dokument werden Design Entscheidungen für Abstimmungen festgehalten und Features erklärt,
+bzw. die Abwesenheit von anderen Features (insbesondere im Vergleich zu Termin Abstimmungen) gerechtfertigt.*
+
```

Comment:
Hier wurde eher Beschrieben wie das System nun aussieht. Jens hatte mal in einer Vorlesung so ungefähr
 gesagt eine Entscheidung ist keine Entscheidung ohne Alternativen. Der Entscheidungsprozess ist viel Interessanter, also das warum. Warum haben wir diese Entscheidungen getroffen